spec/features/editing_topics/show_topics_spec.rb

```diff
@@ -46,7 +46,7 @@ def when_the_document_has_a_topic
   end
 
   def then_i_see_the_topic_breadcrumb
-    within("#topics .topic-breadcrumb") do
+    within("#topics .govuk-list") do
```

Comment:
I think you can drop the .govuk-list specificity here - since that's just used for generic list styling.

Reply:
Done. Within removed

Reply:
the #topics one is still good

Reply:
Sorry @kgarwood I deleted your comment by mistake when I meant to kill one of mine

Reply:
And good idea with an image, the work in this pull request controls rendering of the topics:
<img width="781" alt="content_publisher_pull_525" src="https://user-images.githubusercontent.com/7047139/49018448-5abac700-f183-11e8-99fc-30088562c3a2.png">

Reply:
Fixed, within #topic is retained in the test.